vsts/pipelines/_signbinary.yml

```diff
@@ -0,0 +1,63 @@
+steps:
+- powershell: 'Write-Host "Running Sign Oryx binaries"'
+  displayName: 'Show Build Phase'
+
+- powershell: |
+   Write-Host "##vso[task.setvariable variable=SignType]real"
+   Write-Host "##vso[task.setvariable variable=AGENTBUILD]true"
```

Comment:
q: why do we also set AGENTBUILD here? Isn't signType enough?

Reply:
yes, the way we build the build definitions initially AGENTBUILD was required, now we have enough conditions to have just one variable which is signtype, I wanted to revisit all these cleaning tasks in a separate work-item.